### DISCUSSION TITLE
(Feature request) Auto disable syscalls if signing in to PSN

### DISCUSSION THREAD

**Discussion by ghost** [aldostools/webMAN-MOD#966] [Ideas]
This allows users to sign in to PSN without worry of PSN blocker

**Comment by ghost** [comment]
Also add it to WebMAN Setup if the user wants to uncheck it

**Comment by aldostools** [comment]
This feature will not be implemented the way you describe. There is a similar feature already implemented.

Use R2+TRIANGLE to disable the syscalls. This enable Sign Into PSN.

The Sign Into PSN can be blocked in CFW Evilnat 4.90. This feature was removed from webMAN MOD by default due issues with mappings and conflicts with CFW.

Two weeks ago I added a resource that let install the Sign Into PSN blocker manually
https://github.com/aldostools/Resources/releases/download/Addons/install_custom_npsignin_plugin.pkg
